Yoast/yoast-components
# Added styling for the onboarding wizard.

This pull request adds the following to the onboarding wizard:
- Field descriptions for input fields are no longer headers.
- Text input fields are now aligned next to their input-boxes.
- Title separator choice now has the same look as in the Yoast SEO plugin settings.
- The buttons now have styling.
- When loading the next step, the wizard is greyed out until the next step is loaded.

The next step for this is to separate the components and the CSS for the components.

Testing:
1. Start the application
2. Click next and see if the wizard greys out until the next step is loaded.
3. Check if all the textboxes are displayed correctly and their selected values are remembered when you have switched to another step and return.
4. Go to the title separator choices and see if they are displayed in a fancy way.

NOTE: Custom components like: Google Search Console and Newsletter signup, are not yet implemented, so it’s normal that nothing or weird things are displayed for now.
